var/spack/repos/builtin/packages/podio/package.py

```diff
@@ -105,6 +105,7 @@ class Podio(CMakePackage):
     depends_on("py-tabulate", type=("run", "test"), when="@0.16.6:")
 
     conflicts("+rntuple", when="@:0.16", msg="rntuple support requires at least podio@0.17")
+    conflicts("%gcc@14", when="@:0.99")
```

Comment:
Can we add a reason for why it fails to compile via `msg`?

Reply:
added message for why it files to compile.  0.99 is missing std include files.  These would be easy to add via a patch but the unpatched version will not compile.